Added new category and added CakeManager-plugin

bobmulder opened the pull request:
New Category: 'Admin interface' added.

CakeManager-plugin added to the categories 'Admin interface' and 'Miscellaneous'

dereuromark commented on README.md:
```diff
@@ -82,6 +82,11 @@ Focus is on the specific task.
+## Admin interface
+*Plugins for your admin-area and managing*
+
+- [CakeManager plugin](https://github.com/cakemanager/cakephp-cakemanager) - Manager your app: User-Management, use the admin-area and many components and behaviors available for your app.
```
typo "Manage your app:"

dereuromark commented on README.md:
```diff
@@ -301,6 +306,7 @@ Focus is on the specific task.
 - [Tools plugin](https://github.com/dereuromark/cakephp-tools) - Containing lots of useful libs, helpers, behaviors, components, shells, ...
 - [Travis](https://github.com/FriendsOfCake/travis) - Easy travis setup for CakePHP plugins.
 - [Users plugin](https://github.com/CakeDC/users) - For allowing users to register and login manage their profile.
+- [CakeManager plugin](https://github.com/cakemanager/cakephp-cakemanager) -Basic User-Management system with multiple roles like admin, moderators and normal users.
```
We only list them once. So please remove it here again then.

bobmulder commented:
Okay, you can close this one. I added the plugin to the master, but that was a mistake. I added the plugin to the cake3-branche in an new request. Sorry ;)
